src/test/java/org/broadinstitute/consent/http/resources/DaaResourceTest.java

```diff
@@ -498,7 +518,7 @@ void testDeleteDaaForUserForbidden() {
     libraryCard.setUserId(user.getUserId());
     libraryCard.setDaaIds(List.of(daaId));
 
-    when(userService.findUserByEmail(any())).thenReturn(user);
+    when(userService.findUserByEmail(any())).thenThrow(new ForbiddenException());
```

Comment:
At a technical level, this doesn't reflect what is actually happening. `userService.findUserByEmail()` isn't hit in the resource method that's called in the test. What's actually happening at runtime is that the method is never called in the case that the user doesn't have the correct role - that is controlled by the `UserAuthorizer` class. I think these tests are not necessary here. To actually test this case, I think this would need to run it in a container (see https://www.dropwizard.io/en/release-4.0.x/manual/testing.html#test-containers) but we haven't set up the framework for that yet so it's definitely out of scope for this PR.

Reply:
I was thinking something similar, but then I got confused on how I would be able to test these conditions. I can remove the related tests.